### PR TITLE
Bug 1304753: warn more loudly about garbage secrets

### PR DESCRIPTION
**Pull request by djmitche**

### CODE REVIEW

File: secrets/secreteditor.jsx [hunk 1]
```diff
@@ -105,11 +105,16 @@ var SecretEditor = React.createClass({
...
                  <input type="text" className="form-control" ref="name"
                         placeholder="garbage/<ircnick>/my-secret" />
-                 <p className="text-muted">Note that secrets starting with "garbage/" are visible to just about everybody.  Use them to experiment, but not for real secrets!</p>
+                 <br />
+                 <div className="alert alert-danger">Secrets starting with "garbage/" are visible to just about everybody.  Use them to experiment, but not for real secrets!</div>
                </div>
              ) : (
                 <div className="form-control-static">
                   {this.state.secretId}
+                  { this.state.secretId.startsWith("garbage/") ? (
```
Comment (eliperelman) [inline secrets/secreteditor.jsx]: Just a few things:

- Remove spaces from interpolation braces. With ternaries in JSX, I prefer them to be on their own lines.
- Switch double quotes to single quotes.
- Parens are unnecessary on single-line JSX.
- Switch `undefined` to `null`, and move to the next line:

```jsx
{
  this.state.secretId.startsWith('garbage/') ?
    <div className="alert alert-danger">This is a "garbage" secret and is visible to just about everybody. Do not put any real secrets here!</div> :
    null
}
```

File: secrets/secreteditor.jsx [hunk 2]
```diff
@@ -105,11 +105,17 @@ var SecretEditor = React.createClass({
...
                  <input type="text" className="form-control" ref="name"
                         placeholder="garbage/<ircnick>/my-secret" />
-                 <p className="text-muted">Note that secrets starting with "garbage/" are visible to just about everybody.  Use them to experiment, but not for real secrets!</p>
+                 <br />
+                 <div className="alert alert-danger">Secrets starting with "garbage/" are visible to just about everybody.  Use them to experiment, but not for real secrets!</div>
                </div>
              ) : (
                 <div className="form-control-static">
                   {this.state.secretId}
+									{
```
Comment (eliperelman) [inline secrets/secreteditor.jsx]: Indentation here needs to be moved to the left.

Comment (djmitche) [inline secrets/secreteditor.jsx]: how the heck did I do that :(

Comment (djmitche) [inline secrets/secreteditor.jsx]: oh, tabs, heh -- pushing a fix now

**Pull request merged**